packages/visx-responsive/src/enhancers/withParentSize.tsx

```diff
@@ -9,33 +9,36 @@ interface PrivateWindow {
   ResizeObserver: ResizeObserverPolyfill;
 }
 
-export type WithParentSizeProps = {
```

Comment:
same comment on continuing to export this

Reply:
Responded there 😄